game/static/game/css/board.css

```
@@ -160,30 +160,41 @@ body {
+    padding-bottom: 12px;
 }
 
 .profile-btn{
-    background: #1a1a2e;
-    color: #fff;
-    border: 1px solid #444;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    background: #0f172a;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)